vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/device.go

```diff
@@ -339,7 +339,7 @@ func (device *Device) SupportsETH(chainID uint64) bool {
 	}
 	if device.version.AtLeast(semver.NewSemVer(4, 0, 0)) {
 		switch chainID {
-		case 1, 3, 4: // mainnet, ropsten, rinkeby
+		case 1, 3, 4, 5: // mainnet, ropsten, rinkeby, goerli
```

Comment:
this should be added upstream at https://github.com/digitalbitbox/bitbox02-api-go/. there should be no manual changes inside the vendor/ folder, as they will get lost once a dependency is updated.

support for it is also only possiblr since [v9.10.0](https://github.com/digitalbitbox/bitbox02-firmware/releases/tag/firmware%2Fv9.10.0) - with this version, all chain IDs are supported.

Reply:
My bad, didn't notice that the file was inside vendor! I made a separate PR here https://github.com/digitalbitbox/bitbox02-api-go/pull/71